src/main/resources/default/taglib/w/multiLanguageTextField.html.pasta

```diff
@@ -0,0 +1,173 @@
+<i:arg name="span" type="int" default="6"/>
+<i:arg name="smallSpan" type="int" default="12" description="Defines the span for mobile devices"/>
+<i:arg name="name" type="String" default=""/>
+<i:arg name="fieldName" type="String" default="@name"/>
+<i:arg name="value" type="sirius.db.mixing.types.MultiLanguageString"/>
+<i:arg name="labelKey" type="String" default=""/>
+<i:arg name="label" type="String" default="@i18n(labelKey)"/>
+<i:arg name="forceLabel" type="boolean" default="false"/>
+<i:arg name="helpKey" type="String" default=""/>
+<i:arg name="help" type="String" default="@i18n(helpKey)"/>
+<i:arg name="required" type="boolean" default="false"/>
+<i:arg name="adminOnly" type="boolean" default="false"/>
+<i:arg name="autofocus" type="boolean" default="false"/>
+<i:arg name="type" type="String" default="text"/>
+<i:arg name="id" type="String" default="@('mls-field-' + fieldName)"/>
```

Comment:
generateId() ? namen verwenden ist immer gefährlich...

Reply:
Will be fixed with the JS scope changes